src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div>
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post"
+        class="form form__big"
+        data-qa="big"
+      >
+        <div class="input__border input__border__big">
+          <img
+          src="images/Search.svg"
+          alt="image"
+          class="img__search">
+          <input
+          type="text"
+          class="input__big"
+          placeholder="Try &quot;Los Angeles&quot; "
+          data-qa="keypress" >
```

Comment:
```suggestion
          <input
          type="text"
          class="input__big"
          placeholder="Try &quot;Los Angeles&quot; "
          data-qa="keypress" 
        >
```
closing bracket should be on the same level as opening one